rudderstack/integrations/destinations/destination_adobe_analytics_test.go

```diff
@@ -0,0 +1,27 @@
+package destinations_test
+
+import (
+	"testing"
+
+	cmt "github.com/rudderlabs/terraform-provider-rudderstack/internal/testutil/cm"
+	c "github.com/rudderlabs/terraform-provider-rudderstack/rudderstack/configs"
+)
+
+func TestDestinationResourceAdobeAnalytics(t *testing.T) {
+	cmt.AssertDestination(t, "adobe_analytics", []c.TestConfig{
+		{
+			TerraformCreate: `
+				report_suite_ids = "id001, id002"
+			`,
+			APICreate: `{
+				"reportSuiteIds": "id001, id002"
+			}`,
+			TerraformUpdate: `
+				report_suite_ids = "id001, id003"
```

Comment:
The Update test should ideally include as many config fields as possible. This is in order to ensure that conversion from terraform to API payload is working as expected. The same applies for Create tests. The approach we are usually following with the rest of destinations is to add mandatory fields in the Create test and all fields in the Update test

Reply:
We should also add a Notion link in the PR for compliance purposes (even if this is private)